openapi-validator/src/main/java/com/networknt/openapi/RequestValidator.java

```diff
@@ -176,6 +186,12 @@ private Status validateQueryParameter(final HttpServerExchange exchange,
             if(optional.isPresent()) {
                 return optional.get();
             }
+        } else {
+            Status status = schemaValidator.validate(queryParameterValues, Overlay.toJson((SchemaImpl)queryParameter.getSchema()));
+            Optional<Status> optional = Optional.ofNullable(status);
+            if(optional.isPresent()) {
+                return optional.get();
```

Comment:
And when/why would we be here? What is this section doing?

Reply:
Sure, I will add comments for it as well